sdcm/microbenchmarking.py

```diff
@@ -43,7 +43,7 @@
 LOGGER = logging.getLogger("microbenchmarking")
 LOGGER.setLevel(logging.DEBUG)
 
-MICROBENCHMARK_INDEX_NAME = 'microbenchmarkingv2'
```

Comment:
@temichus is the new type of microbenchmarks the one running withing SCT, share this index ? or they are on a separate index ?

Reply:
@temichus ^^

Reply:
I'm not sure I fully understand the question, but from looking in elasticsearch, all the recent microbnechmarks results are in "microbenchmarkingv2" index.

Reply:
I was talking about the `PerfSimpleQueryAnalyzer`

and seems like it's using a different index coming for configuration:
```
custom_es_index: 'microbenchmarkingtest'
```

but with a very similar name, I was asking to make sure we don't reset those as well at the same time.